handbook/engineering/core-application/playbooks/getting-a-list-of-cloud-users.md

```diff
@@ -0,0 +1,37 @@
+# How to get a list of Sourcegraph Cloud users
+
+This playbook was created as a reference for the future. We expect there might be a need to track how many cloud users we have, how many have used the private repository feature, etc.
+
+This playbook contains SQL queries which should be run against the production Postgres in gcloud. To connect to the production Postgres in gcloud, please use [this guide](../../deployments/postgresql/index.md#sourcegraph-com-specific).
```

Comment:
There seems to be a dashboard in Looker already with similar data. I'm going to change the wording here to make it clearer.

Reply:
Yeah we can run these same queries through BigQuery if needed, happy to share more. We don't pull any private data, but the queries here can all be run since they don't include anything private. Depending on the use case, it may not be sufficient because the data is not live, and updates once per hour.